### DISCUSSION TITLE
Update billing details

### DISCUSSION THREAD

**Discussion by ahpbarreto** [excalidraw/excalidraw#8711] [Q&A]
Hi there,
How can we get a view of all users of our company on Excalidraw so we can update billing details? Do I need to update each account individually or one update for all users? Our credit card is expiring and I don't want to disrupt access to the tool.
cheers
Alex

**Comment by Mrazator** [comment]
Hi! Workspace admin has access to all the members in the workspace and can also update the billing details for the whole workspace.

<img width="212" alt="Screenshot 2024-10-25 at 10 24 17" src="https://github.com/user-attachments/assets/7d6f124f-170c-4a3f-bd2a-bc3dea06d207">

**Comment by Mrazator** [comment]
In case of further questions please ping us at support@excalidraw.com.
